plugins/modules/pnp_intent.py

```diff
@@ -748,15 +776,18 @@ class instance for further use.
                 params={"payload": bulk_list},
                 op_modifies=True,
             )
+            self.log("Devices imported response from the API 'import_devices_in_bulk' is {0}".format(bulk_params), "DEBUG")
```

Comment:
self.log("Response from API 'import_devices_in_bulk' for imported devices: {0}".format(bulk_params), "DEBUG")

Reply:
Resolved